modules/api/pom.xml

```diff
@@ -46,8 +46,8 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-log4j12</artifactId>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j2-impl</artifactId>
```

Comment:
Unfortunately, you'll probably also need to specify log4j-core whenever you specify this one, because of https://issues.apache.org/jira/browse/LOG4J2-3601

Reply:
Interesting that its not a runtime dep. I suppose Fluo was getting it from Accumulo.  I added an explicit dependency in 775237f.

Reply:
Yeah, that's the bug. It's supposed to be a runtime dep... it's a runtime dep for the slf4j 1.7 impl, and it's a runtime dep for the slf4j 1.8 alpha before it got renamed to slf4j 2.0. Unfortunately, somebody in the log4j community is thinking a bit *too* modular-ly and decided to make it an optional dependency, under the idea that somebody mind want want to have the log4j API bind as the impl to slf4j, but then have some other log4j impl bind to the log4j API... it's nonsense, of course, because if they wanted to do that, they'd just bind directly to slf4j. The whole point of the `log4j-slf4j[2]?-impl` jar is to bind to log4j-core. But, somebody lost sight of that when they implemented this one, and it hasn't yet been fixed.

Reply:
> under the idea that somebody mind want want to have the log4j API bind as the impl to slf4j, but then have some other log4j impl bind to the log4j API... it's nonsense,

Interesting, they should wait for that use case to actually materialize and then make the change rather than make the change based on a hypothetical use case that will likely never materialize.